gordo_components/model/factories/feedforward_autoencoder.py

```diff
@@ -86,10 +98,14 @@ def feedforward_model(
         model.add(Dense(units=units, activation=activation))
 
     # Final output layer
+    if isinstance(optimizer, str):
+        Optim = getattr(keras.optimizers, optimizer)
+        optimizer = Optim(**optimizer_kwargs)
+
     model.add(Dense(n_features_out, activation=out_func))
 
     # Set some pre-determined default compile kwargs.
-    compile_kwargs.setdefault("optimizer", "adam")
+    compile_kwargs.update({"optimizer": optimizer})
     compile_kwargs.setdefault("loss", "mean_squared_error")
```

Comment:
Know you didn't write this, but maybe we should default to whatever `Keras` is (ie, not actually set a default), because that's probably where we'll refer people to documentation about optimizer and compile kwargs. If they see the default is something else, that's what I would expect it to be.

Reply:
That sounds proper to me. Should I just go ahead and do that or get a consensus from folks?

Reply:
From [this](https://keras.io/models/sequential/) doc-string it seems like "compile" is a non-optional argument for model.compile, i.e. it does not have a default value. But I think we should have a default, so then adam is maybe as good a default as any?

Reply:
@natalie-caruana @flikka Does "adam" sound good to you all here?

Reply:
Sorry, I was referring to `loss` specifically.

Reply:
It looks like these are [required args](https://keras.io/losses/) for `model.compile`.

Reply:
Adam optimizer as default should be fine.